flutter-intellij-community.iml

```diff
@@ -42,7 +42,25 @@
       </library>
     </orderEntry>
     <orderEntry type="library" name="snakeyaml" level="project" />
-    <orderEntry type="library" name="Dart SDK" level="project" />
-    <orderEntry type="library" name="Dart Packages" level="project" />
+    <orderEntry type="library" exported="" name="Dart SDK" level="project" />
+    <orderEntry type="library" exported="" name="Dart Packages" level="project" />
+    <orderEntry type="module-library">
+      <library>
+        <CLASSES>
+          <root url="jar://$APPLICATION_HOME_DIR$/lib/miglayout-swing-5.0.jar!/" />
```

Comment:
This looks a little brittle to me.  If we're going to depend on a specific jar version would it be safer to put it in a deps dir like was done for `weberknecht-0.1.5.jar`?

Reply:
That would mean distributing the jar with the plugin rather than using the jar distributed with the IDE. @scheglov How was this dependency defined prior to 2018.3?

Reply:
@devoncarew may have ideas but it was just an observation. I'm fine with whatever.

Reply:
> @devoncarew may have ideas but it was just an observation. I'm fine with whatever.

I'd want to make sure that this import works for AS 3.1 / 3.2  and IntelliJ 2017.3 and 2018.1, on the bot and for users in the wild.